documentation/bootstrap_upgrade_notes.md

```diff
@@ -0,0 +1,39 @@
+# Bootstrap upgrade notes
+
+**NOTE: If you have previously disabled your CA, please read this.**
+
+In the past, puppetserver users who have modified their `bootstrap.cfg` file to disable the CA on their compile masters and support a multi-master setup have been subject to a painful upgrade experience.
+
+Starting in puppetserver 2.5.0 this problem will start to be resolved. However, it's important to note that users will have to go through one final annoying upgrade if they're managing the CA service through their `bootstrap.cfg`. This page will walk you through what upgrading will be like if you are managing `bootstrap.cfg` yourself. 
```

Comment:
instead of 'go through one final annoying upgrade', I'd say something like 'users who have disabled the CA service in bootstrap.cfg will need to exercise caution when upgrading to 2.5.0.  Subsequent releases should no longer be subject to this issue.'.